lib/include/public/LogManagerBase.hpp

```diff
@@ -585,6 +585,14 @@ namespace MAT_NS_BEGIN
         static IAuthTokensController* GetAuthTokensController()
             LM_SAFE_CALL_PTR(GetAuthTokensController);
 
+        ///Sets the ticket token with a value
```

Comment:
Put the comment into `<summary></summary>` node.

Reply:
Done